### PR TITLE
Make automated PR build and nightly build to work

### PR DESCRIPTION
**Pull request by Fadis**
…r specific GitHub user.

## Type of PR

- [ ] Changes related to the roadmap (e.g., TODO.md) (type: A) -> Create an issue corresponding to the PR in advance, and refer to this PR in the issue.
- Changes that are not related to the roadmap
    - [ ] Change with multiple possible solutions to the issue (type: B-1) -> Create an issue corresponding to the PR in advance and refer to this PR in the issue.
    - [x] Change with a single solution (type: B-2) -> There is no need to create an issue corresponding to the PR in advance. Please discuss it in this PR.

## Importance of PR

- Importance of the issue
    - [ ] Large (based on several days to weeks of discussion and verification, e.g., this issue is a blocking issue for other issues on the roadmap, etc.)
    - [ ] Medium (based on a few hours to a day of discussion and verification, e.g., this issue is a blocking issue for another minor issue)
    - [x] Small (apparent changes such as build error)
- Complexity of the solution (code, tests, etc.)
    - [ ] Large (requires several days to several weeks of review)
    - [ ] Medium (requires several hours to a day of review)
    - [x] Small (trivial changes, such as build error)

## PR Overview

Since the git repository has moved, Jenkins scripts in the source code is not working properly.
This causes automated PR build on this repository always to fail.
This PR introduce some change on jenkins scripts to make automated PR build and nightly build on this repository working.

Additionary, latest nightly build status is added to the README.md.

## Concerns (Optional)

- [ ] Performance
- [ ] Source Code Quality

---

> The PR author should fill in the following checklist when submitting this PR.

#### Optional Entries
- [ ] If this PR is a PR type A/B-1, there is a cross-link between this PR and the related issue.

#### Mandatory Entries
- [ ] The PR title is a summary of the changes.
- [ ] Completed each required field of the PR.

---
> The PR author should fill out the following checklist in the comments to confirm that this PR is ready to be merged

- [ ] CI is green or confirmed test run results.
- [ ] All change suggestions from reviewers have been resolved (fixed or foregone).

---
> The maintainer of this repository will set up a reviewer for each PR.
> PR reviewers should review this PR in terms of the checklist below before moving on to a detailed code review. Please comment on their initial response by filling in the checklist below.

#### Optional Entries
- [ ] The reviewer assigned more reviewers if needed.
- [ ] The reviewer noted that it is necessary to break out some of the changes in this PR into other PRs if needed.
- [ ] The reviewer noted that the initial response is insufficient if needed.

#### Mandatory Entries
- [ ] The title of this PR summarizes the changes made by this PR properly.
- [ ] The target branch of this PR is as intended.
- [ ] The reviewer understands the issues in this PR.
- [ ] The reviewer plans to review with an appropriate workload based on the importance of this PR.

---
> When the PR reviewer concludes that this PR is ready to be merged, please fill in the checklist below by posting it in the comment. If there is more than one reviewer, please do this on your own.

#### Optional Entries
- [ ] The reviewer noted that if you believe that new tests are needed to evaluate this PR, they have been noted.
- [ ] If minor refactorings are not mentioned in the PR, I understand the intent.
- [ ] If this PR is a PR type A/B-1, we have agreed on this PR's design, direction, and granularity in the related issue.

#### Mandatory Entries
- [ ] The reviewer understands how this PR addresses the issue and the specific changes.
- [ ] This PR uses the best possible issue resolution that the reviewer can think of.
- [ ] This PR is ready to be merged.

### CODE REVIEW

**Comment by Fadis** [comment]
run tests

**Comment by Fadis** [comment]
run tests

File: jenkins/fuzzuf-pr_build_ubuntu1804/Jenkinsfile [hunk 1]
```diff
@@ -13,17 +13,17 @@ pipeline {
...
     }
   }
   post { 
     success {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 18.04でのビルド', true )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 18.04でのビルド', true )
     }
     unsuccessful {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 18.04でのビルド', false )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 18.04でのビルド', false )
```
Comment (RKX1209) [inline jenkins/fuzzuf-pr_build_ubuntu1804/Jenkinsfile:26]: use English here

File: jenkins/fuzzuf-pr_build_ubuntu2004/Jenkinsfile [hunk 2]
```diff
@@ -13,18 +13,18 @@ pipeline {
...
     }
   }
   post { 
     success {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 20.04でのビルド', true )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 20.04でのビルド', true )
     }
     unsuccessful {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 20.04でのビルド', false )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 20.04でのビルド', false )
```
Comment (RKX1209) [inline jenkins/fuzzuf-pr_build_ubuntu2004/Jenkinsfile:27]: use English here

File: jenkins/fuzzuf-pr_build_ubuntu1404/Jenkinsfile [hunk 3]
```diff
@@ -13,17 +13,17 @@ pipeline {
...
     }
   }
   post { 
     success {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 14.04でのビルド', true )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 14.04でのビルド', true )
     }
     unsuccessful {
-      atla_notify_build_result( 'new wyvern', 'Ubuntu 14.04でのビルド', false )
+      atla_notify_build_result( 'fuzzuf', 'Ubuntu 14.04でのビルド', false )
```
Comment (RKX1209) [inline jenkins/fuzzuf-pr_build_ubuntu1404/Jenkinsfile:26]: use English here

**Pull request merged**